### PR TITLE
LandingPage: move Head to _document.js

### PR DESCRIPTION
**Pull request by KarinaDavtyan**
Summary: title was removed due to https://github.com/zeit/next.js/blob/master/errors/no-document-title.md, and global styles are added now as dangerouslySetInnerHTML as in /web

### CODE REVIEW

**Comment by aalises** [comment]
You can still add the title if you move the head to the `_app.js` file (maybe creating a separate `Header` component for the head and styles would be nice). I don't know how important it is right now to have a title for the good ole' SEO...  😆

**Comment by KarinaDavtyan** [comment]
> You can still add the title if you move the head to the `_app.js` file (maybe creating a separate `Header` component for the head and styles would be nice). I don't know how important it is right now to have a title for the good ole' SEO... 😆

Yeah I dont think that adding title is important right now (would be nice to remove placeholders first 😅)

File: apps/landingPage/pages/_document.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
 // @flow
 
 import React from 'react';
-import Document from 'next/document';
+import Document, { Head, Main, NextScript } from 'next/document';
 import { ServerStyleSheet } from 'styled-components';
 
+const globalStyle = {
```
Comment (tbergq) [inline apps/landingPage/pages/_document.js]: any reason we cannot use [createGlobalStyle](https://www.styled-components.com/docs/api#createglobalstyle)?

Comment (KarinaDavtyan) [reply apps/landingPage/pages/_document.js]: I guess not, seem to work fine with it.

**Pull request merged**